lib/cli.ts

```diff
@@ -254,14 +254,20 @@ export = function broccoliCLI(args: string[], ui = new UI()) {
       process.on('SIGTERM', cleanupAndExit);
 
       actionPromise = (async () => {
+        let buildFailed = false;
         try {
           await watcher.start();
         } catch (e) {
+          buildFailed = true;
```

Comment:
Since it seems that the process is already going to die, can we do:

```suggestion
          process.exitCode = 1;
```

Instead (then we don't need either of the `process.exit` calls directly)..

Reply:
Yup it makes sense. TIL process.exitCode.